addons/project/models/project_project.py

```diff
@@ -26,9 +26,13 @@ def _compute_task_count(self):
         project_and_state_counts = self.env['project.task'].with_context(
             active_test=any(project.active for project in self)
         )._read_group(
-            [('project_id', 'in', self.ids)],
-            ['project_id', 'state'],
-            ['__count'],
+            domain=[
```

Comment:
Me personally, I prefer it with keywords, as I find it more readable. But @xavierbol, didn't you tell me it was more efficient without?

Reply:
@auon-odoo 
TBH, that's what I heard but I don't know the gain and I don't know if it is true or not.

And so, I would say you can do what you prefer. 🙂